{"uuid":"34964b05-7c7a-4357-a29e-4fe092f67d03","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["60692ce4-f883-470b-8855-abb6354cd2dc"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681589120980,"stop":1681589120980},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681589121208,"stop":1681589121208},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681589121294,"stop":1681589121294},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681589121355,"stop":1681589121356},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681589121371,"stop":1681589121372}],"start":1681589120876,"stop":1681589121372}